### ISSUE TITLE
Dev Lite Test: PhET Desktop App 2023-06-05

### ISSUE THREAD

**Issue by jbphet** [phetsims/qa#954]
@kathy-phet, @oliver-phet, and @KatieWoe:  There is a new update of the PhET Desktop App for Windows and MacOS available at https://phet-dev.colorado.edu/weddell/dist/2023-06-05/.  This was rebuilt based on a request from @RVieyra, and the primary motivation for updating is to pick up the most recent sims and a bunch of translations that have been submitted since the last build, which was from May of 2022.

QA, please:

- [ ] Verify that Greenhouse Effect, Calculus Grapher, Geometric Optics, and any other sims that you're aware of that have been published for the first time after May of 2022 are present in the sim.
- [ ] Verify that some sims are present for the `tw`, `sw`, and `az` locales and that they run correctly.  You don't need to test _all_ sims for these locales, just a handful.  These are locales for which there have been a lot of translations submitted over the last year, so the objective here is to make sure that the desktop app has picked up the recent translations.

Testing:
- [ ] Tester = , Platform = , Time = 
- [ ] Tester = , Platform = , Time =

**Comment by Nancy-Salpepi** [comment]
@jbphet, it has been a while since I tested the desktop app, but this version doesn't seem to be up to date. The most recent sims are missing (https://github.com/phetsims/weddell/issues/108) and the About Dialogs for the available sims list a version date from March. Every sim I checked has an available update. 
I also compared the number of translated sims in az. On the website, only 4 are listed as "Not Yet Translated" while on the app the number is 50+.
